services/web-app/.storybook/styles.module.scss

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* stylelint-disable declaration-no-important */
+/* stylelint-disable declaration-property-value-disallowed-list */
```

Comment:
Should we put comments next to these indicating why they're needed in this file?